dev/com.ibm.ws.anno/resources/com/ibm/ws/anno/resources/internal/AnnoMessages.nlsprops

```diff
@@ -139,15 +139,15 @@ ANNO_CLASSSOURCE_JAR_STATE_BAD.useraction=CMSG0002
 # # {1} The name of a jar file
 ANNO_CLASSSOURCE_CLOSE5_EXCEPTION=CWWKC0014W: [ {0} ]: The close of [{1}] failed.
 ANNO_CLASSSOURCE_CLOSE5_EXCEPTION.explanation=An attempt to close the JarFile object caused an exception to occur.
-ANNO_CLASSSOURCE_CLOSE5_EXCEPTION.useraction=See the exception and the associated FFDC message for more information.
+ANNO_CLASSSOURCE_CLOSE5_EXCEPTION.useraction=For more information on the topic, see the exception and the associated FFDC message.
 
 # # {0} A hashcode id used to identify the object emitting the message
 # # {1} a resource name
 # # {2} a class name
 # # {3} a jar file path
 ANNO_CLASSSOURCE_OPEN5_EXCEPTION=CWWKC0015W: [ {0} ]: The open of [{1}] for class [{2}] in [{3}] failed.
 ANNO_CLASSSOURCE_OPEN5_EXCEPTION.explanation=There was an entry detected for the class in the jar file, but the creation of an InputStream for the class caused an exception.
-ANNO_CLASSSOURCE_OPEN5_EXCEPTION.useraction=See the exception and the associated FFDC message for more information.
+ANNO_CLASSSOURCE_OPEN5_EXCEPTION.useraction=For more information on the topic, see the exception and the associated FFDC message.
```

Comment:
Change
'For more information on the topic, see the exception and the associated FFDC message.'
to
'For more information, see the exception and the associated FFDC message.'

Reply:
Updated.